src/PhpSpec/Listener/ClassNotFoundListener.php

```diff
@@ -75,7 +76,13 @@ public function afterSuite(SuiteEvent $event)
 
             if ($this->io->askConfirmation($message)) {
                 $this->generator->generate($resource, 'class');
-                $event->markAsWorthRerunning();
+                if (class_exists($classname)) {
+                    $event->markAsWorthRerunning();
+                }
+                else {
```

Comment:
You could lose this else by throwing the exception if the class doesn't exist first.